.github/issue_template.md

```diff
@@ -0,0 +1,37 @@
+<!---
```

Comment:
Was this file added by mistake?

Reply:
Yep, master is ahead of develop for this file. Fixed.